packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomLabels xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
After installing the unlocked package are users able to edit the labels ?

Reply:
Yes they can edit the labels and translations